.github/scripts/setup-env.sh

```diff
@@ -81,7 +52,7 @@ fi
 echo '::endgroup::'
 
 echo '::group::Install PyTorch'
-pip install --progress-bar=off --pre torch --index-url="${PYTORCH_WHEEL_INDEX}"
+eval "${PIP_INSTALL_TORCH}"
```

Comment:
This is coming from [here](https://github.com/pytorch/test-infra/blob/ffb9897d2f4652ef2d84c2142b49f7d47211a0c7/tools/pkg-helpers/pytorch_pkg_helpers/wheel.py#L15) and for example [used in the wheel build workflows](https://github.com/pytorch/test-infra/blob/ffb9897d2f4652ef2d84c2142b49f7d47211a0c7/.github/workflows/build_wheels_linux.yml#L101) as well.

Reply:
@malfet This uses [`--extra-index-url` unconditionally](https://github.com/pytorch/test-infra/blob/ffb9897d2f4652ef2d84c2142b49f7d47211a0c7/tools/pkg-helpers/pytorch_pkg_helpers/wheel.py#L27). Shouldn't that be `--index-url` for nightly and test channels?

Reply:
> This is coming from [here](https://github.com/pytorch/test-infra/blob/ffb9897d2f4652ef2d84c2142b49f7d47211a0c7/tools/pkg-helpers/pytorch_pkg_helpers/wheel.py#L15) and for example [used in the wheel build workflows](https://github.com/pytorch/test-infra/blob/ffb9897d2f4652ef2d84c2142b49f7d47211a0c7/.github/workflows/build_wheels_linux.yml#L101) as well.

Unfortunately, this is not part of the general setup. It is done in the [`setup-binary-builds` action](https://github.com/pytorch/test-infra/blob/ffb9897d2f4652ef2d84c2142b49f7d47211a0c7/.github/actions/setup-binary-builds/action.yml#L115).